sdk/python/feast/inference.py

```diff
@@ -29,9 +29,17 @@ def update_entities_with_inferred_types_from_feature_views(
         col_names_and_types = view.batch_source.get_table_column_names_and_types(config)
         for entity_name in view.entities:
             if entity_name in incomplete_entities:
+                entity = incomplete_entities[entity_name]
+                entity_table_col_join_key = (
```

Comment:
Nit: the name `entity_table_col_join_key` seems a bit verbose. What about just `entity_join_key`?

Reply:
Done